docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -0,0 +1,736 @@
+    "strapi-hook-knex": "3.0.0-beta.x",
+    "strapi-utils": "3.0.0-beta.x"
+  }
+}
+```
+
+Starting from this new release plugins are now npm dependencies so, you need to install them too.
+
+Here are the default ones. If you have installed other plugins you will also need to add them here (e.g `strapi-plugin-graphql`).
+
+```json
+{
+  "dependencies": {
+    //...
+    "strapi-plugin-content-manager": "3.0.0-beta.x",
+    "strapi-plugin-content-type-builder": "3.0.0-beta.x",
+    "strapi-plugin-email": "3.0.0-beta.x",
+    "strapi-plugin-settings-manager": "3.0.0-beta.x",
+    "strapi-plugin-upload": "3.0.0-beta.x",
+    "strapi-plugin-users-permissions": "3.0.0-beta.x"
+  }
+}
+```
+
+#### `Scripts`
+
+We have completely refreshed the `scripts` of a project so update them as follows:
+
+**Before**
+
+```json
+{
+  "scripts": {
+    "setup": "cd admin && npm run setup",
+    "start": "node server.js",
+    "strapi": "node_modules/strapi/bin/strapi.js",
+    "postinstall": "node node_modules/strapi/lib/utils/post-install.js"
+    //...
+  }
+}
+```
+
+**After**
+
+```json
+{
+  "scripts": {
+    "develop": "strapi develop",
+    "start": "strapi start",
+    "build": "strapi build",
+    "strapi": "strapi"
+    //...
+  }
+}
+```
+
+::: warning
+
+To avoid confusions we have decided to give the same behaviour to the `npm run start` and `strapi start` commands. To do so we had to introduce a new `strapi develop` command to run your project in `watch` mode.
+
+:::
+
+#### `Strapi`
+
+We removed the need for the `packageManager` key under `strapi`.
+
+**Before**
+
+```json
+{
+  "strapi": {
+    //...
+    "packageManager": "yarn"
+  }
+}
+```
+
+**After**
+
+```json
+{
+  "strapi": {
+    //...
+  }
+}
+```
+
+### Installing your new dependencies
+
+You can now run
+
+```bash
+npm install
+# or
+yarn
+```
+
+## Migrating `server.js`
+
+As of now, strapi can start without a `server.js` file. You can now delete it.
+
+If you need a server.js file to be able to run `node server.js` instead of `npm run start` then create a `./server.js` file as follows:
+
+```js
+const strapi = require('strapi');
+
+strapi(/* {...} */).start();
+```
+
+## Migrating `config`
+
+You can leave all your files in `./config` unchanged and remove the `server.autoReload` key in `./config/environments/**/server.json`.
+
+## Migrating `plugins`
+
+One of our main objectives for the `beta` is to make it easier and quicker to upgrade to more recent versions of Strapi. This is why starting from now, plugins will be located in the `node_modules` folder.
+
+[Read more](https://strapi.io/documentation/3.0.0-beta.x/concepts/concepts.html#files-structure)
+
+### Migrating non customized plugin
+
+If you installed a plugin but never modified any file inside `./plugins/pluginName/**/*`, you can remove the `./plugins/pluginName` folder.
+
+### Migrating customized plugin
+
+If you have made some modifications in one of your plugins you will have to do some manual migrations.
```

Comment:
If you have made some modifications **to** one of your plugins; you will have to do some manual migrations: